README.md

```diff
@@ -94,5 +94,15 @@ If you have any suggestions about ways that we can improve how we use this tool,
 
 ## FAQ
 
+#### CSS Variables
 [How do I use CSS variables to customize
 the library?](https://pf4.patternfly.org/guidelines#variables)
+
+#### Browser Support
```

Comment:
Should we also have a section for device support?

Reply:
I think we should leave device support out, we should endeavor to support all devices that run the latest versions of the specified browsers.